test/base.js

```diff
@@ -14,6 +14,8 @@ var Promise = require('pinkie-promise');
 var yeoman = require('yeoman-environment');
 var userHome = require('user-home');
 
+var LF = process.platform === 'win32' ? '\r\n' : '\n';
```

Comment:
also, http://blog.npmjs.org/post/145260155635/introducing-hooks-get-notifications-of-npm offers a lot of interesting possibilities